### PR TITLE
[Doc] Fix code examples syntax errors

### PR DESCRIPTION
**Pull request by WiXSL**

### CODE REVIEW

File: docs/AuthRBAC.md [hunk 1]
```diff
@@ -894,17 +894,17 @@ const authProvider = {
 const ProductEdit = () => (
     <Edit>
         <AccordionForm>
-            <AccordionForm.Panel label="description" label="Description">
+            <AccordionForm.Panel label="Description">
```
Comment (djhi) [inline docs/AuthRBAC.md:897]:
```suggestion
            <AccordionForm.Panel name="description" label="Description">
```

File: docs/AuthRBAC.md [hunk 2]
```diff
@@ -894,17 +894,17 @@ const authProvider = {
...
                 <TextInput source="reference" />
                 <TextInput source="width" />
                 <TextInput source="height" />
                 <TextInput source="description" />
             </AccordionForm.Panel>
-            <AccordionForm.Panel label="images" label="Images">
+            <AccordionForm.Panel label="Images">
```
Comment (djhi) [inline docs/AuthRBAC.md:903]:
```suggestion
            <AccordionForm.Panel name="images" label="Images">
```

File: docs/AuthRBAC.md [hunk 3]
```diff
@@ -894,17 +894,17 @@ const authProvider = {
...
                 <TextInput source="image" />
                 <TextInput source="thumbnail" />
             </AccordionForm.Panel>
-            <AccordionForm.Panel label="stock" label="Stock">
+            <AccordionForm.Panel label="Stock">
```
Comment (djhi) [inline docs/AuthRBAC.md:907]:
```suggestion
            <AccordionForm.Panel name="stock" label="Stock">
```

File: docs/AuthRBAC.md (hunk 1 above)
Comment (djhi) [reply docs/AuthRBAC.md:897]: This actually required for permissions

**Review by djhi — CHANGES_REQUESTED** [review]

File: docs/Menu.md [hunk 4]
```diff
@@ -336,7 +337,7 @@ import { Admin, Layout, LayoutProps, Resource } from 'react-admin';
 import { MenuLive } from '@react-admin/ra-realtime';
 import { PostList, PostShow, PostEdit, realTimeDataProvider } from '.';
 
-const CustomLayout = (props: LayoutProps) => (
+const CustomLayout = (props) => (
```
Comment (djhi) [inline docs/Menu.md:340]: Please change the codefence language instead. Same where possible.

**Comment by WiXSL** [comment]
Back to RFR

**Review by adguernier — APPROVED** [review]

**Review by slax57 — APPROVED** [review]
Incredible work, thank you so much! :muscle: :pray:

**Pull request merged**